clients/pkg/promtail/server/server.go

```diff
@@ -51,6 +51,7 @@ type Config struct {
 	ExternalURL       string `yaml:"external_url"`
 	HealthCheckTarget *bool  `yaml:"health_check_target"`
 	Disable           bool   `yaml:"disable"`
+	Debug             bool   `yaml:"debug"`
```

Comment:
I wonder if we should find a better name that reflect the behaviour.

There's other endpoint available for profiling under /debug/fgprof and this doesn't disable it. Should it ?

Reply:
I didn't realize there was another endpoint under `/debug/`, but yes, I think it should be disabled as well. I chose this rather generic option name precisely because I figured there might be some other debugging facility (now or in the future) that should most likely not be running in production by default.

Reply:
But I see the issue. The other profiling endpoint is `/debug/pprof`, which gets registered in a vendored dependency (https://github.com/grafana/loki/blob/main/vendor/github.com/weaveworks/common/server/server.go#L391-L397), along with `/metrics`.

So there isn't an easy way to disable `/debug/pprof` without disabling `/metrics` as well.

Reply:
I've added a commit to deal with the `/debug/pprof` endpoint. Let me know what you think.